recipes/pytest-codspeed/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "pytest-codspeed" %}
+{% set version = "2.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytest_codspeed-{{ version }}.tar.gz
+  sha256: 665003fc20117b64a98d16ffd1008f5bd6bf3b1e9af142b98c00abff7f626bbd
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - hatchling
+    - pip
+  run:
+    - python >=3.7
+    - cffi >=1.15.1,<1.16.dev0
+    - pytest >=3.8
+    - filelock >=3.12.2,<3.13.dev0
+    - setuptools >=67.8.0,<67.9.dev0
```

So conda-forge doesn't have `setuptools=67.8.0` since they closed the PR - https://github.com/conda-forge/setuptools-feedstock/pull/317, causing the build to fail:

```
WARNING: failed to get install actions, retrying.  exception was: Unsatisfiable dependencies for platform linux-64: {MatchSpec("setuptools[version='>=67.8.0,<67.9.dev0']")}
Encountered problems while solving:
  - nothing provides requested setuptools >=67.8.0,<67.9.dev0

Could not solve for environment specs
The following package could not be installed
└─ setuptools >=67.8.0,<67.9.dev0  does not exist (perhaps a typo or a missing channel).
```

Maybe wait for upstream to loosen this requirement - https://github.com/CodSpeedHQ/pytest-codspeed/pull/21#discussion_r1437248655

---

Pin can be removed, xref https://github.com/CodSpeedHQ/pytest-codspeed/blob/v2.2.1/pyproject.toml#L41. Or we could even remove this completely since it is only needed to build cffi on Python 3.12?
```suggestion
    - setuptools
```